modules/cluster/main.tf

```diff
@@ -68,18 +72,6 @@ resource "google_container_node_pool" "default" {
   node_config {
     machine_type    = var.cluster_compute_machine_type
     service_account = var.service_account.email
-    oauth_scopes = [
-      "https://www.googleapis.com/auth/bigtable.admin",
-      "https://www.googleapis.com/auth/bigtable.data",
-      "https://www.googleapis.com/auth/bigquery",
-      "https://www.googleapis.com/auth/cloud-platform",
-      "https://www.googleapis.com/auth/devstorage.read_write",
-      "https://www.googleapis.com/auth/logging.write",
-      "https://www.googleapis.com/auth/monitoring",
-      "https://www.googleapis.com/auth/pubsub",
-      "https://www.googleapis.com/auth/trace.append",
-      "https://www.googleapis.com/auth/sqlservice.admin",
-    ]
```

Comment:
Are these just not needed? Wondering why we'd want to remove these.

Reply:
GCP doesn't recommend using those. I tried removing them and nothing happened so it seems like the IAM permissions are sufficient.

> We recommend that you don't specify your own access scopes. Instead, you should use [IAM permissions](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating) or [Kubernetes role-based access control (RBAC)](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control) to grant access to specific IAM service accounts or Kubernetes service accounts.
Ref: https://cloud.google.com/kubernetes-engine/docs/how-to/access-scopes#what_are_access_scopes